README.md

```diff
@@ -69,6 +69,55 @@ options for you and run the image using `podman`:
     ~/src/flathub/com.example.App/com.example.App.json
 ```
 
+### On Flathub
+
+Flathub runs this tool hourly for all Flatpak repos under [github.com/flathub](https://github.com/flathub). This means for those repos to receive update PRs, simply ensure any `x-checker-data` you need is present [as described below](https://github.com/flathub/flatpak-external-data-checker#changes-to-flatpak-manifests). Note Flathub's hosted tool only checks the default branch.
+
+To stop Flathub's tool from checking your repo, add `"disable-external-data-checker": true` to `flathub.json` in the default branch.
+
+### Custom workflow
+
+Alternatively, you can use own workflow. This can be useful if e.g. wanting to update non-default branches.
+
+Put this yaml file under `.github/workflows`, e.g. put it in `.github/workflows/update.yaml`. Ensure to put the correct path to the manifest in the last line.
+
+<details><summary>GitHub update workflow</summary><br/>
```

Comment:
I don't think we need to collapse the snippets. The readme is already long enough and requires ToC to navigate, so the snippets won't change much.

Reply:
Yeah, I agree it doesn't help much with readme length. Removed the collapsing.